pkg/bootstrapper/bootstrapper.go

```diff
@@ -481,107 +489,37 @@ func (wmcb *winNodeBootstrapper) createKubeletService() error {
 		StartType:    mgr.StartAutomatic,
 		ErrorControl: 0,
 		// Path to kubelet.exe
-		BinaryPathName:   filepath.Join(wmcb.installDir, "kubelet.exe"),
-		LoadOrderGroup:   "",
-		TagId:            0,
-		Dependencies:     nil,
+		BinaryPathName: filepath.Join(wmcb.installDir, "kubelet.exe"),
+		LoadOrderGroup: "",
+		TagId:          0,
+		// set dependency on docker
+		Dependencies:     []string{"docker"},
```

Comment:
Is this service going to be named `docker` across cloud providers / Windows images?

Reply:
The name docker is independent of cloud providers, the package management msft used to install docker package on any Windows server images, installs them with service named "docker"